certif/app/routes/authenticated/sessions/details.js

```diff
@@ -1,7 +1,17 @@
 import Route from '@ember/routing/route';
 
 export default class SessionsDetailsRoute extends Route {
-  model(params) {
-    return this.store.findRecord('session', params.session_id);
+  async model(params) {
+    const session = await this.store.findRecord('session', params.session_id);
+    await session.certificationCandidates;
+    const certificationCenter = this.modelFor('authenticated');
+    const featureToggles = this.store.peekRecord('feature-toggle', 0);
+    const isCertifPrescriptionScoEnabled = featureToggles.certifPrescriptionSco;
+
+    return {
+      session,
+      isCertificationCenterSco: certificationCenter.isSco,
+      isCertifPrescriptionScoEnabled,
+    };
```

Comment:
@D0C702WH0 @aceol @sroccaserra @Anne-Gaelle-S @EGI-OCTO 
Note for ourselves : il faut impérativement que les tests passent à chaque étape, on a fait ça il y a 3 jours, on a laissé les tests cassés jusqu'à la fin et là ça fait 1 jour et demi qu'on galère à les réparer.

Reply:
Oui, l'idéal serait de faire passer les tests au fur et à mesure.
Après la problématique qu'on avait eu c'était que les tests ne passaient pas mais le fonctionnel ne marchait pas non plus (et là côté front c'est dur je trouve de vouloir faire du test first). Du coup, on s'est d'abord occupé de faire marcher le fonctionnel et je vous ait lâchement abandonné dès que ça a fonctionné 😅.

Reply:
Comme disais Bob Marley, no tests no cry